Jenkinsfile

```diff
@@ -31,3 +42,37 @@ node('linux') {
 			sh 'make test'
 	}
 }
+
+def git_commit = ''
+def go_list = ''
+
+stage 'Go & Git Info'
+node('linux') {
+
+	def gopath = gopath_linux()
+	def workdir = workdir_linux(gopath)
+
+	dir(workdir) {
+		checkout scm
+
+		// HACK: Recommended approach for getting command output is writing to and then reading a file.
+		sh 'mkdir -p tmp'
+		sh 'go list . > tmp/GO_LIST'
```

Comment:
update: currently using env var `CHANGE_AUTHOR` to swap in for `deis` in the go pkg/repo name...